### PR TITLE
Double size limit to 140kb

### PR DESCRIPTION
**Pull request by ahuth**
### Summary:

We're now over the 70kb size limit from the size-limit-action. This PR doubles it to 140kb, since it's expected that the package will keep getting bigger.

I'm not sure we should even use this action. But maybe it's nice to get the report on how much size increases per PR.

### Test Plan:

- CI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,13 +32,13 @@
...
       "path": "packages/components/lib/**/*([a-zA-Z-_]).js",
-      "limit": "70KB",
+      "limit": "140KB",
       "name": "components",
       "webpack": false
     },
     {
       "path": "packages/components/lib/**/*([a-zA-Z-_]).css",
-      "limit": "70KB",
+      "limit": "140KB",
```
Comment (ahuth) [inline package.json:41]: I'm not sure this config is right.

It seems to indicate that JS and CSS should each be able to be 70kb in size. But the CI failures (for example, see #816) seem like it's the JS + CSS that's putting us over 70kb.

File: package.json [hunk 2]
```diff
@@ -32,13 +32,13 @@
   "size-limit": [
     {
       "path": "packages/components/lib/**/*([a-zA-Z-_]).js",
-      "limit": "70KB",
+      "limit": "140KB",
```
Comment (ahuth) [inline package.json:35]: Unrelated to this change, but why are we using a forked version of size-limit-action? 🤔 

Something to think about for the future is seeing if we can get the non-forked version working for us.

Comment (ahuth) [reply package.json:35]: Ah, looks like Devin added some code to not post more than 1 comment from the action: https://github.com/chanzuckerberg/size-limit-action/commit/afba6038fb5322674ca9669c5482a47aac7a31c0

**Review by lesliecodes — APPROVED** [review]

**Comment by ahuth** [comment]
Thanks!

**Pull request merged**